sdk/python/tests/utils/auth_permissions_util.py

```diff
@@ -0,0 +1,244 @@
+import os
+
+import yaml
+from keycloak import KeycloakAdmin
+
+from feast import (
+    Entity,
+    FeatureStore,
+    FeatureView,
+    OnDemandFeatureView,
+    RepoConfig,
+    StreamFeatureView,
+)
+from feast.infra.registry.remote import RemoteRegistryConfig
+from feast.permissions.action import AuthzedAction
+from feast.permissions.permission import Permission
+from feast.permissions.policy import RoleBasedPolicy
+from tests.utils.cli_repo_creator import CliRunner
+
+REMOTE_REGISTRY_SERVER_PROJECT_NAME = "test_registry_server"
+
+list_permissions_perm = Permission(
```

Comment:
are you planning to use the same permissions in the IT code? Otherwise I don't see the need for adding these instances in the `utils` common folder